homeassistant/setup.py

```diff
@@ -281,6 +281,19 @@ async def _async_setup_component(
         integration = await loader.async_get_integration(hass, domain)
     except loader.IntegrationNotFound:
         _log_error_setup_error(hass, domain, None, "Integration not found.")
+        ir.async_create_issue(
+            hass,
+            HOMEASSISTANT_DOMAIN,
+            f"integration_not_found.{domain}",
+            is_fixable=True,
+            issue_domain=domain,
```

Comment:
Not sure which issue_domain to put it under. Otherwise it would show the `integration not found` placeholder most of the time

Reply:
I suggest we keep `homeassistant`  as issue domain.